assignments/callbacks.js

```diff
@@ -2,48 +2,74 @@
 
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
-/* 
+
 
   //Given this problem: 
   
-  function firstItem(arr, cb) {
-    // firstItem passes the first item of the given array to the callback function.
-  }
-
+  // function firstItem(arr, cb) {
+  //   // firstItem passes the first item of the given array to the callback function.
+  // }
   // Potential Solution:
-
   // Higher order function using "cb" as the call back
   function firstItem(arr, cb) {
     return cb(arr[0]);
   }
-
   // Function invocation 
   firstItem(items, function(first) {
     console.log(first)
   });
 
-*/
-
 
+// Problem #1---------------------------------------------------
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+
+  return cb(arr.length);
 }
+getLength(items, function(length){
+  console.log(length);
+});
 
+// Problem #2---------------------------------------------------
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length-1]);
 }
+last(items, function(last){
+  console.log(last);
+});
+
 
+// Problem #3---------------------------------------------------
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x,  y);
 }
 
+function add(x, y){
+  return x + y;
+}
+console.log(sumNums(3, 7, add));
+
+
+
+// Problem #4---------------------------------------------------
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x, y);
 }
 
+function multiply(x, y){
+  return x * y;
+}
+console.log(multiplyNums(3, 7, multiply));
+
+
+// Problem #5---------------------------------------------------
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
```

Comment:
So there is a built in method called includes that you can utilize here to see if something is present in a given dataset. If could look like this inside the block, return cb(list.includes(item) ? true : false);    note the use of a ternary operator with the question mark and the colon, this is a short way of writing an if else statement. To the left of the question mark goes the condition, in this case asking if list.includes(item). To the right of the question mark is what is output if the condition passes and to the right of the colon is what is output if the condition does not pass